src/components/ContextMenu.vue

```diff
@@ -0,0 +1,48 @@
+<script lang="ts">
+import store from '@/store';
+import { computed } from '@vue/composition-api';
+
+export default {
+  setup() {
+    const rightClickMenu = computed(() => store.getters.rightClickMenu);
```

Comment:
It doesn't seem like this component should depend on values stored in global state. It seems a lot like this component ought to manage its own position values, and use a `v-model` pointing to a boolean stored in the component where the menu will appear. I found a nice article explaining how `v-model` works, and how it can be used in custom components: https://dev.to/vue-storefront/vue-during-coffee-break-using-v-model-with-custom-components-3bo9.

Reply:
See https://github.com/multinet-app/multilink/issues/203